app/_hub/kong-inc/grpc-gateway/index.md

```diff
@@ -0,0 +1,158 @@
+---
+name: gRPC-gateway
+publisher: Kong Inc.
+
+categories:
+  - transformations
+alpha: true
+type: plugin
+
+desc: Access gRPC services through HTTP REST
+description: |
+  A Kong plugin to allow access to a gRPC service via HTTP REST requests
+  and translate requests and responses in a JSON format. Similar to 
+  [gRPC-gateway].
```

Comment:
@fffonion is this supposed to be gRPC web? If this is supposed to be a link, it isn't working and it mentions itself.

Reply:
updated, thanks!